can't install dependency yargs in docker

Why did the `yargs` dependency change from `"yargs": "github:NickCarneiro/yargs"` to `"yargs": "git@github.com:NickCarneiro/yargs-ansi.git"`? My docker is erroring out now:

```bash
#20 20.53 npm ERR! prepareGitDep npm ERR! Error while executing:
#20 20.53 npm ERR! prepareGitDep npm ERR! /usr/bin/git ls-remote -h -t ssh://git@github.com/NickCarneiro/yargs-parser-ansi.git
#20 20.53 npm ERR! prepareGitDep npm ERR!
#20 20.53 npm ERR! prepareGitDep npm ERR! Host key verification failed.
#20 20.53 npm ERR! prepareGitDep npm ERR! fatal: Could not read from remote repository.
```

--- a/bin/cli.js
+++ b/bin/cli.js
@@ -1,7 +1,7 @@
 #!/usr/bin/env node
 
 const curlconverter = require('../index.js')
-const argv = require('yargs').argv
+const argv = require('@curlconverter/yargs').argv
 const fs = require('fs')
 
 // sets a default incase -l/--langauge isn't passed

--- a/package.json
+++ b/package.json
@@ -1,6 +1,6 @@
 {
   "name": "curlconverter",
-  "version": "3.20.0",
+  "version": "3.21.0",
   "description": "convert curl syntax to native python and javascript http code",
   "homepage": "https://github.com/NickCarneiro/curlconverter",
   "author": {
@@ -25,8 +25,7 @@
     "query-string": "^7.0.1",
     "string.prototype.startswith": "^1.0.0",
     "yamljs": "^0.3.0",
-    "yargs": "^17.2.1",
-    "yargs-parser": "git@github.com:curlconverter/yargs-parser.git"
+    "@curlconverter/yargs": "^0.0.2"
   },
   "devDependencies": {
     "standard": "^16.0.4",

--- a/test.js
+++ b/test.js
@@ -4,7 +4,7 @@ const fs = require('fs')
 
 const utils = require('./util')
 const curlconverter = require('./index.js')
-const yargs = require('yargs')
+const yargs = require('@curlconverter/yargs')
 
 // the curl_commands/ directory contains input files
 // The file name is a description of the command.

--- a/util.js
+++ b/util.js
@@ -1,5 +1,5 @@
 const cookie = require('cookie')
-const yargs = require('yargs')
+const yargs = require('@curlconverter/yargs')
 const URL = require('url')
 const querystring = require('query-string')
 const nunjucks = require('nunjucks')

